internal/utils/testutil/binary.go

```diff
@@ -24,15 +24,27 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-func GinkgoGetHorusecCmd(horusecCmd string) *exec.Cmd {
+type HorusecCmd string
+
+func (h HorusecCmd) String() string {
+	return string(h)
+}
+
+const (
```

Comment:
isn't this constants the same as [this](https://github.com/ZupIT/horusec/blob/ba4a6a37cc79a63a830e8761db666a185cdc7ad9/internal/utils/testutil/cli_args.go#L18) ?

Reply:
adjusted!